ios/FluentUI/Popup Menu/PopupMenuController.swift

```diff
@@ -257,7 +292,7 @@ open class PopupMenuController: DrawerController {
     }
 
     private func initTableView() {
-        tableView.backgroundColor = backgroundColor
+        tableView.backgroundColor = .clear
```

Comment:
Did you test overrides?

Reply:
Yep those work!
![image](https://github.com/microsoft/fluentui-apple/assets/22566866/fc5c6331-50d4-48d4-ba99-335fd76862e2)
![image](https://github.com/microsoft/fluentui-apple/assets/22566866/87e5b8af-9ea8-4e63-a94c-2d5b3d252646)